cabal.project

```diff
@@ -1,2 +1,438 @@
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: ouroboros-consensus/ouroboros-consensus-test-infra
+
+source-repository-package
+    type: git
+    location: https://github.com/input-output-hk/ouroboros-network
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: ouroboros-network
+
+source-repository-package
+    type: git
+    location: https://github.com/input-output-hk/ouroboros-network
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: ouroboros-network-framework
+
+source-repository-package
+    type: git
+    location: https://github.com/input-output-hk/ouroboros-network
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: ouroboros-network-testing
+
+source-repository-package
+    type: git
+    location: https://github.com/input-output-hk/ouroboros-network
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: typed-protocols
+
+source-repository-package
+    type: git
+    location: https://github.com/input-output-hk/ouroboros-network
+    tag: ec74d1c463b6d18bdca2b714abe730c1d237cc32
+    subdir: typed-protocols-examples
+
+source-repository-package
+    type: git
+    location: https://github.com/well-typed/cborg
+    tag: 42a83192749774268337258f4f94c97584b80ca6
+    subdir: cborg
+
+source-repository-package
+    type: git
+    location: https://github.com/snoyberg/http-client.git
+    tag: 1a75bdfca014723dd5d40760fad854b3f0f37156
+    subdir: http-client
+
+allow-older: *
+allow-newer: *
+
+package cardano-crypto
+  tests: False
+
+package contra-tracer
+  tests: False
+
+package iohk-monitoring
+  tests: False
+
+package zip
+  tests: False
+  flags: +disable-bzip2
+
+package cardano-crypto-praos
+  tests: False
+  flags: -external-libsodium-vrf
+
+package byron-spec-ledger
+  tests: False
+
+package ouroboros-consensus-cardano
+  tests: False
+
+package small-steps
+  tests: False
+
+package small-steps-test
+  tests: False
+
+package ouroboros-consensus
+  tests: False
+
+package ouroboros-consensus-byron
+  tests: False
+
+package ouroboros-consensus-shelley
+  tests: False
+
+package ouroboros-consensus-mock
+  tests: False
+
+package cborg
+  tests: False
+
+package http-client
+  tests: False
+
+package goblins
+  tests: False
+
+package io-sim
+  tests: False
+
+package io-sim-classes
+  tests: False
+
+package shelley-spec-non-integral
+  tests: False
+
+package network-mux
+  tests: False
+
+package typed-protocols
+  tests: False
+
+package typed-protocols-examples
+  tests: False
+
+package cardano-prelude
+  tests: False
+
+package cardano-binary
+  tests: False
+
+package cardano-sl-x509
+  tests: False
+
+package cardano-crypto-class
+  tests: False
+
+package cardano-crypto-wrapper
+  tests: False
+
+package ouroboros-network-framework
+  tests: False
+
+package ouroboros-network
+  tests: False
+
+-- force systemd off until there's a solution to
+-- https://github.com/haskell/cabal/issues/5444
+flags: -systemd
```

Comment:
This is being debated in https://github.com/input-output-hk/cardano-node/pull/1666 as well